sdk/identity/identity-broker/src/index.ts

```diff
@@ -1,10 +1,23 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import type { AzurePluginContext } from "../../identity/src/plugins/provider";
```

Comment:
We should not pulling in files from outside the package and I fixed that by:

- Copying a subset of the interface exposed internally in Identity, so that we do not need to reach into the bowels of Identity to pull it
- Copying the `msalNodeTestSetup` files locally
- Depending on `@azure/identity` instead of a relative path in tests